app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayViewModel.kt

```diff
@@ -47,4 +46,15 @@ class NavigationOverlayViewModel(
     fun shouldBeDisplayed(channelDetails: Observable<ChannelDetails>): Observable<Boolean> =
         channelDetails.map { it.tileList.isNotEmpty() }
             .distinctUntilChanged()
+
+    val focusView : Observable<Int> = screenController.currentActiveScreen
+            .buffer(2,1)
+            .filter{ (_, currentScreen) -> currentScreen == ActiveScreen.NAVIGATION_OVERLAY }
+            .map { (prevScreen, _) ->
+                when(prevScreen) {
+                    ActiveScreen.WEB_RENDER -> R.id.navUrlInput
+                    ActiveScreen.SETTINGS -> R.id.settings_tile_telemetry
+                    else -> View.NO_ID
```

Comment:
Could you replace the `else` with `ActiveScreen.NAVIGATION_OVERLAY`?  That way if we add any screens in the future, this will be a compilation error and we'll know to update it.  With the `else` branch it would just work, which could lead to bad behavior when `View.NO_ID` was emitted.

Reply:
I'm getting a Kotlin compiler warning (`Enum argument can be null in Java, but exhaustive when contains no null branch`) when I remove the else case. Any way I can get around this?